packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -123,6 +125,8 @@ export default defineComponent({
     pattern: { type: String },
     inputmode: { type: String, default: 'text' },
     ariaLabel: { type: String, default: undefined },
+    counter: { type: Boolean, default: false },
```

Comment:
How about moving it all in one prop?

```suggestion
    counter: { type: [Boolean, Number], default: false },
```